.github/actions/image-summary/action.yml

```diff
@@ -70,9 +70,9 @@ runs:
             digest="$(crane digest --platform=linux/amd64 "${ref}:${primary}" || crane digest "${ref}:${primary}")"
             cosign download attestation \
               --predicate-type https://spdx.dev/Document \
-              "${ref}@${digest}" | head -1 | jq -r .payload | base64 -d | jq -M -c .predicate > sbom.json
+              "${ref}@${digest}" | jq -r .payload | base64 -d | jq -M -c .predicate > sbom.json
 
-            version="$(cat sbom.json | jq -r '.packages[] | select(.name == "'${pkg}'") | .versionInfo')"
+            version="$(cat sbom.json | jq -r '.packages[] | select(.name == "'${pkg}'") | .versionInfo' | uniq)"
```

Comment:
The `uniq` should deduplicate and leave the single version they should all have since this happens due to reproducible builds getting multiple attestations.